components/cam/src/physics/cam/nudging.F90

```diff
@@ -231,6 +240,97 @@ module nudging
 !      Nudge_File_Ntime    - INT  Number of time slices per nudging data file
 !    /
 !
+! A typical namelist setting of nudged global simulation (UV-only, tau = 6h) 
+! with this revised nudging code is provided below:
+!
+!    &nudging_nl
+!      Nudge_Model         = .True.
+!      Nudge_Path          = 'PATH_TO_DATA'
+!      Nudge_File_Template = 'E3SM.cam.h1.%y-%m-%d-00000.nc'
+!      Nudge_Times_Per_Day = 4            ! nudging input data frequency
+!      Model_Times_Per_Day = 48           ! should not be larger than 48 if dtime = 1800s
+!      Nudge_Uprof         = 1
+!      Nudge_Ucoef         = 1.
+!      Nudge_Vprof         = 1
+!      Nudge_Vcoef         = 1.
+!      Nudge_Tprof         = 0
+!      Nudge_Tcoef         = 0.
+!      Nudge_Qprof         = 0
+!      Nudge_Qcoef         = 0.
+!      Nudge_PSprof        = 0
+!      Nudge_PScoef        = 0.
+!      Nudge_Beg_Year      = 0000         ! begin year of nudged simulation
+!      Nudge_Beg_Month     = 1            ! begin month of nudged simulation
+!      Nudge_Beg_Day       = 1            ! begin day of nudged simulation
+!      Nudge_End_Year      = 9999         ! end year of nudged simulation
+!      Nudge_End_Month     = 1            ! end month of nudged simulation
+!      Nudge_End_Day       = 1            ! end day of nudged simulation
+!      Nudge_Method        = 'Linear'     ! use linear-interpolation nudging
+!      Nudge_File_Ntime    = 4            ! if there are four time slices per nudging input data file 
+!      Nudge_Loc_Same      = .TRUE.
+!    /
+!
+! If a user prefers one time slice per nudging input data file, simply change
+! two settings above to the following settings:
+!
+!      Nudge_File_Template = 'E3SM.cam.h1.%y-%m-%d-%s.nc'
+!      Nudge_File_Ntime    = 1
+!
+! More advanced example settings for the CONUS RRM simulation (i.e., Table 3 of Tang et al. (2019), 
+! using UV-only, tau=6h, four time slices per nudging input data file, nudging spatial window) 
+! with this revised nudging code are:
+!
+!    &nudging_nl
+!      Nudge_Model         = .true.
+!      Nudge_Path          = 'PATH_TO_DATA'
+!      Nudge_File_Template = ‘interim_se_%y%m%d00_%y%m%d18_TQUV-%s.nc’
```

Comment:
This file template is for 1 time slice per file because of "-%s". It should be deleted to be consistent with Nudge_File_Ntime    = 4.

Reply:
Thanks @tangq. I have just updated the comments in the nudging.F90 about this part.

Reply:
Thanks, @tangq . I have just deleted the "-%s" and updated the comments in the nudging.F90.

Reply:
Thank you, @jiansunpnnl, for making the changes.

Reply:
No problem. Thanks for pointing out this issue.